src/main/resources/localstack/init-aws.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+aws s3 --endpoint-url http://localhost:5001 mb s3://test-optout-bucket
+aws s3 --endpoint-url http://localhost:5001 cp /s3/optout/optout/ s3://test-optout-bucket/ --recursive
```

Comment:
I assume you're doing this for a reason but I can't figure out why

Reply:
This was a workaround I did with splitting the data between uid2-admin and uid2-optout, but I can switch `/s3/optout/optout/` back to `/s3/optout/`